jdbc/src/main/spotbugs/spotbugs-exclude.xml

```diff
@@ -18,4 +18,9 @@
     <Class name="software.amazon.timestream.jdbc.TimestreamTablesResultSet"/>
     <Method name="populateCurrentRows"/>
   </Match>
+  <Match>
+    <Bug pattern="SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE"/>
```

Comment:
```suggestion
    <!--The exclude is added as currently driver does not support the prepared statement. This is not a new exclude, as it has already been added for populateCurrentRows for TimestreamTablesResultSet in this file.-->
    <Bug pattern="SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE"/>
```

Reply:
Done. Thanks! Didn't think of adding a comment in the exclude file before.